### PR TITLE
Add ocs-monkey test

### PR DESCRIPTION
**Pull request by jilju**
Added a test to run ocs-monkey, a chaos testing suite for OCS.
https://github.com/red-hat-storage/ocs-monkey

Signed-off-by: Jilju Joy <jijoy@redhat.com>

### CODE REVIEW

File: tests/ocs-monkey/test_ocs_monkey.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import time
+
+from ocs_ci.framework.testlib import ignore_leftovers
+from ocs_ci.utility.utils import clone_repo, run_cmd
+from ocs_ci.ocs import constants
+
+log = logging.getLogger(__name__)
+
+
+@ignore_leftovers
```
Comment (ebenahar) [inline tests/ocs-monkey/test_ocs_monkey.py:14]: @hnallurv @PrasadDesala should this test be part of `workloads` tier?

Comment (RazTamir) [reply tests/ocs-monkey/test_ocs_monkey.py:14]: IMO it should be in a new tier level.
As this is not a real workload we can try to integrate it with the other workloads that are running for a long time.
I'm thinking mostly about the BDI workload that can run for hours and with the OCS monkey in place, we are really taking the test to the next level

Comment (ebenahar) [reply tests/ocs-monkey/test_ocs_monkey.py:14]: yes. I have discussed this with @jilju and he brought up the idea of having a dedicated Jenkins job specifically for ocs-monkey test

**Review by ebenahar — APPROVED** [review]

**Review by AviadP — APPROVED** [review]

File: tests/ocs-monkey/test_ocs_monkey.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+def test_ocs_monkey():
+    ocs_monkety_dir = '/tmp/ocs-monkey'
+    # ocs-monkey run time in seconds
+    run_time = 3200
+    clone_repo(constants.OCS_MONKEY_REPOSITORY, ocs_monkety_dir)
```
Comment (mykaul) [inline tests/ocs-monkey/test_ocs_monkey.py:19]: Why not get OCS monkey as a git submodule of OCS-CI?

**Pull request merged**